nixos/modules/programs/firejail.nix

```diff
@@ -20,13 +19,38 @@ let
     '';
   };
 
+  wrappedPkgs = map (pkg:
+    pkgs.stdenv.mkDerivation {
+      name = "firejail-" + pkg.name;
+      buildCommand = ''
+        mkdir -p $out
+        ln -s ${pkg}/* $out
+        rm $out/bin
```

Comment:
What if the package doesn't have a `bin` directory? Doesn't make much sense in combination with firejail, but might happen. In that case there should probably be a useful error message.